doc/ssh.md

```diff
@@ -0,0 +1,44 @@
+# Getting SSH access
+
+See [special access to buildresources](./process/special_access_to_build_resources.md)
+for the detailed policy on access to machines.
+
+If you are not a member of the build working group, open an issue in
+the build repo for temporary SSH keys. The following instructions are
+for build working group members.
+
+1. If you have not added your public keys in the [secrets repo][],
+   follow the instructions there to gain access to the credentials.
+2. Copy the SSH keys in the [secrets repo][] to your `~/.ssh` folder. What keys
```

Comment:
It might be good to add a section about adding a personal password to the unencrypted keyfiles with
```
ssh-keygen -p -f keyfile
```